src/main/java/org/opentripplanner/standalone/GraphBuilderParameters.java

```diff
@@ -258,6 +265,7 @@ public GraphBuilderParameters(JsonNode config) {
         banDiscouragedWalking = config.path("banDiscouragedWalking").asBoolean(false);
         banDiscouragedBiking = config.path("banDiscouragedBiking").asBoolean(false);
         maxTransferDistance = config.path("maxTransferDistance").asDouble(2000);
+        generateWheelchairAccessibleTransfers = config.path("generateWheelchairAccessibleTransfers").asBoolean(true);
```

Comment:
I'd recommend setting this as false by default so as to not automatically turn it on for trip planners that aren't processing wheelchair-accessible trip plans.

```suggestion
        generateWheelchairAccessibleTransfers = config.path("generateWheelchairAccessibleTransfers").asBoolean(false);
```